src/table.c

```diff
@@ -4,60 +4,90 @@
-82, 86, 92, 97, 103, 108, 114, 121, 127, 134, 141, 149, 156, 164, 172, 181, 190, 199, 209, 219, 229, 239, 250, 262, 273,
-285, 298, 311, 324, 338, 352, 366, 381, 397, 413, 429, 446, 464, 482, 500, 519, 538, 559, 579, 600, 622, 644, 667, 691, 715,
-740, 765, 791, 818, 845, 873, 902, 931, 961, 992, 1024, 1056, 1090, 1123, 1158, 1194, 1230, 1267, 1305, 1344, 1383, 1424, 1465,
-1508, 1551, 1595, 1640, 1686, 1733, 1781, 1830, 1880, 1931, 1983, 2036, 2090, 2146, 2202, 2259, 2318, 2377, 2438, 2500, 2563, 2627,
-2693, 2760, 2827, 2897, 2967, 3039, 3112, 3186, 3262, 3339, 3417, 3497, 3578, 3660, 3744, 3829, 3916, 4005, 4094, 4185, 4278,
-4373, 4468, 4566, 4665, 4765, 4868, 4971, 5077, 5184, 5293, 5403, 5516, 5630, 5745, 5863, 5982, 6104, 6227, 6351, 6478, 6607,
-6737, 6870, 7004, 7140, 7279, 7419, 7561, 7706, 7852, 8000, 8151, 8304, 8459, 8616, 8775, 8936, 9100, 9266, 9434, 9604, 9777, 9952,
-10129, 10309, 10491, 10675, 10862, 11051, 11243, 11437, 11634, 11833, 12035, 12240, 12447, 12656, 12869, 13083, 13301, 13521,
-13744, 13970, 14199, 14430, 14664, 14901, 15141, 15384, 15629, 15878, 16129
+    0,     0,     0,     0,     0,     0,     0,     0,     0,     0,     0,
+    0,     0,     0,     0,     0,     0,     0,     0,     0,     1,     1,
+    1,     1,     1,     1,     2,     2,     2,     3,     3,     4,     4,
+    5,     5,     6,     6,     7,     8,     9,     10,    11,    12,    13,
+    14,    16,    17,    19,    20,    22,    24,    26,    28,    30,    32,
+    35,    38,    40,    43,    46,    49,    53,    56,    60,    64,    68,
+    72,    77,    82,    86,    92,    97,    103,   108,   114,   121,   127,
+    134,   141,   149,   156,   164,   172,   181,   190,   199,   209,   219,
+    229,   239,   250,   262,   273,   285,   298,   311,   324,   338,   352,
+    366,   381,   397,   413,   429,   446,   464,   482,   500,   519,   538,
+    559,   579,   600,   622,   644,   667,   691,   715,   740,   765,   791,
+    818,   845,   873,   902,   931,   961,   992,   1024,  1056,  1090,  1123,
+    1158,  1194,  1230,  1267,  1305,  1344,  1383,  1424,  1465,  1508,  1551,
+    1595,  1640,  1686,  1733,  1781,  1830,  1880,  1931,  1983,  2036,  2090,
+    2146,  2202,  2259,  2318,  2377,  2438,  2500,  2563,  2627,  2693,  2760,
+    2827,  2897,  2967,  3039,  3112,  3186,  3262,  3339,  3417,  3497,  3578,
+    3660,  3744,  3829,  3916,  4005,  4094,  4185,  4278,  4373,  4468,  4566,
+    4665,  4765,  4868,  4971,  5077,  5184,  5293,  5403,  5516,  5630,  5745,
+    5863,  5982,  6104,  6227,  6351,  6478,  6607,  6737,  6870,  7004,  7140,
+    7279,  7419,  7561,  7706,  7852,  8000,  8151,  8304,  8459,  8616,  8775,
+    8936,  9100,  9266,  9434,  9604,  9777,  9952,  10129, 10309, 10491, 10675,
+    10862, 11051, 11243, 11437, 11634, 11833, 12035, 12240, 12447, 12656, 12869,
+    13083, 13301, 13521, 13744, 13970, 14199, 14430, 14664, 14901, 15141, 15384,
+    15629, 15878, 16129
 };
```

Comment:
Ah, wait, I see it in `.editorconfig`.  Maybe this file just needs to be reprocessed.